config/flog.yml

```diff
@@ -1,2 +1,2 @@
 ---
-threshold: 27.7
+threshold: 33.8
```

Comment:
Flay / Flog scores are not to be interpreted absolute, only relative to the code change. Ideally both grow logarithmic to the amount of complexity.